mavros/src/plugins/imu.cpp

```diff
@@ -328,18 +328,18 @@ class IMUPlugin : public plugin::PluginBase {
 		/** MAVLink quaternion exactly matches Eigen convention.
 		 *  The RPY describes the rotation: aircraft->NED.
 		 *  It is required to change this to aircraft->base_link:
-		 *  @snippet src/plugins/imu.cpp ned->baselink->enu
+		 *  @snippet src/plugins/imu.cpp ned->baselink->nwu
 		 */
-		auto enu_baselink_orientation = ftf::transform_orientation_aircraft_baselink(
+		auto nwu_baselink_orientation = ftf::transform_orientation_aircraft_baselink(
```

Comment:
actually this is `enu_baselink_orientation` - check what is applied upfront. The orientation should be on ENU frame. Raw IMU data is in the NWU frame (ROS body frame).